src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Events/EventPage.razor

```diff
@@ -35,7 +35,7 @@
         var entity = await _behind.Read(default);
         if (entity == null)
         {
-            throw GuardHelper.Exception($"Event cannot be null here");
+           GuardHelper.ThrowIfNull(entity, "Event cannot be null here");    
         }
```

Comment:
Does this make sense to you? Did you look inside the definition of `ThrowIfNull`?